test/Spago/Build.purs

```diff
@@ -100,6 +100,23 @@ spec = Spec.around withTempDir do
         }
       spago [ "build" ] >>= shouldBeFailure
 
+    Spec.it "should censor warnings with given errorcode and prefix messsage" \{spago, fixture} -> do 
+       spago [ "init" ] >>= shouldBeSuccess
+       let 
+           srcMain = Path.concat [ "src" , "Main.purs" ] 
+           spagoYaml = "spago.yaml"
+       FSA.unlink srcMain 
+       FS.copyFile
+          { src : fixture "censor-warnings.purs" 
+          , dst : srcMain
+          } 
+       FSA.unlink spagoYaml 
+       FS.copyFile 
+        { src : fixture "censor-warnings.yaml"
+        , dst : spagoYaml
+        }
+       spago [ "build" ] >>= shouldBeSuccess
+
```

Comment:
A few notes:
- if you move the fixture files into a directory that is structures as a spago project, then you can use `FS.copyTree` to just copy the project in the test directory, so you don't have to call `init` and do this file juggling. See [the fixtures for the monorepo tests for an example](https://github.com/purescript/spago/tree/master/test-fixtures/monorepo)
- the test is not checking that things are properly censored, just asserting that the build succeeds. We should check that we are not getting these warnings - see this test in the monorepo suite that is doing a similar thing:

https://github.com/purescript/spago/blob/fdaa7e63b1357cc7309db75a18d99421756de13b/test/Spago/Build/Monorepo.purs#L124-L135

Reply:
okay this one looks neat, will do this👍

Reply:
Made the changes in the test cases, can you check once?